ui/src/app/machines/views/MachineDetails/MachineStorage/constants.ts

```diff
@@ -0,0 +1,3 @@
+// From models/partition.py. This should ideally be available over the websocket.
+// https://github.com/canonical-web-and-design/maas-ui/issues/1866
+export const MIN_PARTITION_SIZE = 4 * 1024 * 1024;
```

Comment:
Should this possibly be in store/machine? Would we ever need to reference it outside of the storage tab?

Also, you could use a TypeScript enum for this value.

Reply:
Possibly, but what I actually want to do is get rid of this constant from the UI code entirely because at the moment it has to be manually kept in sync with the backend.